### PR TITLE
build: make our integration tests report clear steps for CircleCI UI

### PR DESCRIPTION
**Pull request by fntlnz**
inspection via collect test data [0]

[0] https://circleci.com/docs/2.0/collect-test-data/

Signed-off-by: Lorenzo Fontana <lo@linux.com>

**What type of PR is this?**

/kind cleanup

/kind documentation

**Any specific area of the project related to this PR?**

/area build

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/run_regression_tests.sh [hunk 1]
```diff
@@ -104,8 +104,11 @@ function run_tests() {
         suites+=($SCRIPTDIR/falco_tests_package.yaml)
     fi
     
+    mkdir -p ${OPT_BUIL_DIR}/integration-tests-xunit
```
Comment (leodido) [inline test/run_regression_tests.sh]:
```suggestion
    mkdir -p ${OPT_BUILD_DIR}/integration-tests-xunit
```

You probably mean `OPT_BUILD_DIR` here.

Comment (fntlnz) [reply test/run_regression_tests.sh]: Thank you!

**Review by leogr — DISMISSED** [review]

**Review by leogr — APPROVED** [review]

**Comment by fntlnz** [comment]
W00t!
image

**Review by leodido — APPROVED** [review]

**Pull request merged**